src/app/inventory/store/item-index.ts

```diff
@@ -11,7 +11,8 @@ export function createItemIndex(item: DimItem): string {
   // Try to make a unique, but stable ID. This isn't always possible, such as in the case of consumables.
   let index = item.id;
   if (item.id === '0') {
-    _idTracker[index] = (_idTracker[index] || 0) + 1;
+    _idTracker[index] ||= 0;
+    _idTracker[index]++;
```

Comment:
Could also be `(_idTracker[index] ||= 0)++` if you wanted to be evil

Reply:
unfortunately no, you can access a self-editing method of something, like `(thing??=[]).push(`
but increment and assignment operators want something more valid on the left